### DISCUSSION TITLE
FTP extension no longer in docker images `8.2.15` and `8.3.2`; update README

### DISCUSSION THREAD

**Discussion by ToshY** [mlocati/docker-php-extension-installer#870] [General]
After pulling `8.3.2` I noticed that one of my applications was having issues as one of it's dependencies required `ext-ftp`, but it turns out that for `8.2.15` and `8.3.2` the `ftp` extension is no longer enabled by default. Issue can be found in [docker-library/issues/1488](https://github.com/docker-library/php/issues/1488).

<details><summary><code>docker run --rm php:8.3.1-cli php -m</code></summary>

```shell
[PHP Modules]
Core
ctype
curl
date
dom
fileinfo
filter
ftp
hash
iconv
json
libxml
mbstring
mysqlnd
openssl
pcre
PDO
pdo_sqlite
Phar
posix
random
readline
Reflection
session
SimpleXML
sodium
SPL
sqlite3
standard
tokenizer
xml
xmlreader
xmlwriter
zlib

[Zend Modules]
```

</details> 

<details><summary><code>docker run --rm php:8.3.2-cli php -m</code></summary>

```shell
[PHP Modules]
Core
ctype
curl
date
dom
fileinfo
filter
hash
iconv
json
libxml
mbstring
mysqlnd
openssl
pcre
PDO
pdo_sqlite
Phar
posix
random
readline
Reflection
session
SimpleXML
sodium
SPL
sqlite3
standard
tokenizer
xml
xmlreader
xmlwriter
zlib

[Zend Modules]
```

</details> 

Maybe an additional note or update of README is in order.

**Comment by mlocati** [comment]
I've just fixed this issue with (see #871): since the version 2.1.78 of `install-php-extensions`, you can simply install the FTP extension with SSL support by using

```
install-php-extensions ftp
```

**Reply by ToshY** [reply]
Thanks for the info (and the work)!
